main/test/ca/uwaterloo/flix/language/phase/TestEntryPoints.scala

```diff
@@ -189,6 +189,222 @@ class TestEntryPoints extends AnyFunSuite with TestUtils {
     val result = compile(input, Options.TestWithLibMin.copy(entryPoint = Some(Symbol.mkDefnSym("f"))))
     expectError[EntryPointError.MainEntryPointNotFound](result, allowUnknown = true)
   }
+  test("Test.DefaultHandlerNotInEffectModule.01") {
+    val input =
+      """
+        |pub eff TestEff{
```

Comment:
Rename:

- TestEff to just E
- doSomething to op

(Everywhere). This allows one to focus more on the essentials.